.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           go-version: ${{ matrix.go }}
 
       - name: go vet
-        if: ${{ !startsWith(matrix.os, 'windows-') && !startsWith(matrix.os, 'macos-') }}
+        if: ${{ !startsWith(matrix.os, 'macos-') && !startsWith(matrix.os, 'windows-') }}
```

Comment:
This change is not related to the issue. I just wanted to reorder them in an alphabetical order. I can make it in a later PR.

Reply:
Reverted this change.